rest/data/subscription.go

```diff
@@ -45,8 +114,34 @@ func (dc *DBSubscriptionConnector) GetSubscriptions(owner string, ownerType even
 	return apiSubs, nil
 }
 
-func (dc *DBSubscriptionConnector) DeleteSubscription(id string) error {
-	return event.RemoveSubscription(id)
+func (dc *DBSubscriptionConnector) DeleteSubscriptions(owner string, ids []string) error {
+	for _, id := range ids {
+		subscription, err := event.FindSubscriptionByID(id)
+		if err != nil {
+			return gimlet.ErrorResponse{
+				StatusCode: http.StatusInternalServerError,
+				Message:    err.Error(),
+			}
+		}
+		if subscription == nil {
+			return gimlet.ErrorResponse{
+				StatusCode: http.StatusNotFound,
+				Message:    "Subscription not found",
+			}
+		}
+		if subscription.Owner != owner {
```

Comment:
How does this work with subscriptions at the project level? Is there a concept of an owner there?

Reply:
At the project level, the owner is the project ID (I also verified this by checking some documents in staging).